ionic.config.json

```diff
@@ -5,6 +5,6 @@
   "typescript": true,
   "proxies": [{
     "path": "/",
-    "proxyUrl": "http://localhost:4000"
+    "proxyUrl": "http://linkupmobile.eternussolutions.com/webapi"
```

Comment:
@MaheshNikate : Please change it to localhost URL. We should avoid publishing production URL if possible.